### PR TITLE
[Hub Generated] Review request for Microsoft.Web to add version stable/2019-08-01

### PR DESCRIPTION
**Pull request by miwebst**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/miwebst/azure-rest-api-specs/dev-web-Microsoft.Web-2019-08-01-staticsitespl...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by miwebst** [comment]
@weidongxu-microsoft This PR is failing with breaking changes because we are adding a property to the response of the staticSites objects. This API has been published to swagger but is behind a whitelist until we preview, thus there won't be any breaking changes. Can you help us manually skip this test or provide a suggestion?

Also seeing that all other tests have passed, can you request ARM feedback?

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json [hunk 1]
```diff
@@ -1449,6 +1449,13 @@
         "buildProperties": {
           "$ref": "#/definitions/StaticSiteBuildProperties",
           "description": "Build properties to configure on the repository."
+        },
+        "privateEndpointConnections": {
```
Comment (weidongxu-microsoft) [inline specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json:1453]: Add non-readonly property in stable is risky for breaking change.

**Comment by weidongxu-microsoft** [comment]
@miwebst

ARM feedback requested.

Do you mean currently this service is not exposed to customer?

**Comment by miwebst** [comment]
@weidongxu-microsoft thanks! Yes the service is currently behind a whitelist flag so its not being actively used by customers. That being said, this field is readonly so I've made that change :) Thanks for the help!

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json [hunk 2]
```diff
@@ -1449,6 +1449,14 @@
...
+          "description": "Private endpoint connections referencing this resource.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/PrivateEndpointConnectionResource"
+          },
+          "readOnly": true
         }
```
Comment (pilor) [inline specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json:1460]: if staticSites supports privateEndpointConnections it should expose them as child resources as well and have a privateLinkResources collection GET as a child

**Review by pilor — CHANGES_REQUESTED** [review]
:clock1:

**Comment by weidongxu-microsoft** [comment]
A reference definition could be found here https://github.com/Azure/azure-rest-api-specs/pull/8935/files

There is effort to move it to common types.

**Comment by ravbhatnagar** [comment]
This is a very old PR and the comments from ARM review have not been responded/addressed. Removing the ARM review label. PLease re-add if this PR still needs review.

**Pull request closed without merging**